client/asset/zec/shielded_test.go

```diff
@@ -0,0 +1,136 @@
+//go:build shielded
+
+package zec
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"decred.org/dcrdex/client/asset"
+	"decred.org/dcrdex/dex"
+	dexzec "decred.org/dcrdex/dex/networks/zec"
+)
+
+var (
+	dextestDir  = filepath.Join(os.Getenv("HOME"), "dextest")
+	harnessDir  = filepath.Join(dextestDir, "zec", "harness-ctl")
+	ctx         = context.Background()
+	log         = dex.StdOutLogger("ZT", dex.LevelTrace)
+	tBtcParams  = dexzec.RegressionNetParams
+	tAddrParams = dexzec.RegressionNetAddressParams
+)
+
+func harnessCmd(ctx context.Context, exe string, args ...string) (string, error) {
+	cmd := exec.CommandContext(ctx, exe, args...)
+	cmd.Dir = harnessDir
+	op, err := cmd.CombinedOutput()
+	return string(op), err
+}
+
+func getDeltaWallet(t *testing.T) (*zecWallet, func()) {
+	wi, err := NewWallet(&asset.WalletConfig{
+		Type: walletTypeRPC,
+		Settings: map[string]string{
+			"rpcuser":     "user",
+			"rpcpassword": "pass",
+			"rpcport":     "33770",
```

Comment:
I successfully ran the tests with the `alpha` wallet, but when running with the other wallets I got 
```
rawrequest (getwalletinfo) error: -28: This wallet operation is disabled while reindexing.
```

Might just be an issue on my end.

Reply:
That error usually means that a node has not finished its initial sync. Non-alpha zec harness nodes take a while to get caught up after you run the harness. Is that possible?

Reply:
Could be, but it was definitely running for a while before I ran the tests. I'll try again though.

Reply:
On the beta node I saw `This wallet operation is disabled while reindexing` on my first manual RPC to the wallet (a `z_getbalanceforaccount`) even though it had been running for quite a long time.  I wonder if we need to give each node a "tick" from the harness shell script somehow?

Reply:
No luck tweaking zec/harness.sh to fix this long delay.  Hopefully we find a solution, but we might have to live with it.